### PR TITLE
Implemented integration test for the offline installer

### PR DESCRIPTION
**Pull request by Naatan**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-1295" title="DX-1295" target="_blank">StoryDX-1295</a>  Offline installers have integration tests
  </td></table>
  <br />

### CODE REVIEW

File: internal/fileutils/fileutils.go [hunk 1]
```diff
@@ -949,6 +959,19 @@ func ListDirSimple(sourcePath string, includeDirs bool) []string {
 	return result
 }
 
+// ListFilesUnsafe lists filepaths under the given sourcePath non-recursively
+func ListFilesUnsafe(sourcePath string) []string {
+	result := []string{}
+	files, err := ioutil.ReadDir(sourcePath)
+	if err != nil {
+		panic(err)
```
Comment (mitchell-as) [inline internal/fileutils/fileutils.go:967]: In other panics, we add a sprintf message. Would that be helpful here?

File: internal/subshell/cmd/env.go [hunk 2]
```diff
@@ -85,7 +85,7 @@ func cleanPath(keyValue, oldEntry string) string {
 }
 
 // setUserEnv sets a variable in the user environment and saves the original as a backup
```
Comment (mitchell-as) [inline internal/subshell/cmd/env.go:87]: This comment needs updating.

File: test/integration/offinstall_int_test.go [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+		}
+		tp.Expect("TEST REPLACEMENT", 5*time.Second)
+		tp.ExpectExitCode(0)
+	}
+
+	{ // Uninstall
+		tp := ts.SpawnCmdWithOpts(
+			suite.uninstallerPath,
+			e2e.WithArgs(targetDir),
+			e2e.AppendEnv("ACTIVESTATE_CLI_DISABLE_RUNTIME=false"),
```
Comment (mitchell-as) [inline test/integration/offinstall_int_test.go:89]: Does the uninstaller need a runtime enabled?

Comment (Naatan) [reply test/integration/offinstall_int_test.go:89]: Yes, it needs to access the runtime in order to find out what files to delete and whatnot.

File: activestate.yaml [hunk 4]
```diff
@@ -68,6 +68,9 @@ scripts:
       export GO111MODULE=on
       goflags="${GOFLAGS}"; unset GOFLAGS
       GOFLAGS="${goflags}"
+      pushd ~
+      go install github.com/sanderhahn/gozip/cmd/gozip@latest
+      popd
```
Comment (shaunmlowry) [inline activestate.yaml:73]: 1. This should be vendored (or at least not using latest) so the build is reproducible
2. You should be using our fork of gozip from here: https://github.com/ActiveState/gozip

**Review by shaunmlowry — CHANGES_REQUESTED** [review]

File: activestate.yaml (hunk 4 above)
Comment (Naatan) [reply activestate.yaml:73]: Why do we have a fork? We've made no changes to it.

**Review by mitchell-as — DISMISSED** [review]
This looks fine from my perspective.

**Review by mitchell-as — DISMISSED** [review]

**Pull request merged**